src/Component/tests/StateMachine/OperationStateMachineTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Tests\StateMachine;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface;
+use Sylius\Resource\Context\Context;
+use Sylius\Resource\Metadata\Create;
+use Sylius\Resource\Metadata\Index;
+use Sylius\Resource\Metadata\StateMachineAwareOperationInterface;
+use Sylius\Resource\StateMachine\OperationStateMachine;
+use Sylius\Resource\StateMachine\OperationStateMachineInterface;
+
+final class OperationStateMachineTest extends TestCase
+{
+    private ContainerInterface $locator;
+
+    private OperationStateMachine $operationStateMachine;
+
+    protected function setUp(): void
+    {
+        $this->locator = $this->createMock(ContainerInterface::class);
+        $this->operationStateMachine = new OperationStateMachine($this->locator);
+    }
+
+    public function testItIsInitializable(): void
+    {
+        $this->assertInstanceOf(OperationStateMachine::class, $this->operationStateMachine);
+    }
```

Comment:
This can be removed as it would crash in the setup method if that was not the case.